optional-integration-analysis/perform-integration.R

```diff
@@ -0,0 +1,85 @@
+## This script performs integration on processed SCE objects that have been
+## merged in the `merge-sce.R` script
+
+## Set up ----------------------------------------------------------------------
+
+# Load libraries
+library(optparse)
+library(dplyr)
+library(SingleCellExperiment)
+library(scpcaTools)
+
+# Declare command line options
+option_list <- list(
+  make_option(
+    opt_str = c("--merged_sce_rds"),
```

Comment:
```suggestion
    opt_str = c("--merged_sce_file"),
```
To keep things consistent with the output and how we usually set up the input lets use `file`.